builtin/mainmenu/tab_about.lua

```diff
@@ -117,11 +117,16 @@ return {
 			buildCreditList(previous_contributors) .. "," ..
 			";1]"
 
+		-- Render information
+		fs = fs .. "label[0.75,4.9;" ..
+			fgettext("Active render:") .. "\n" ..
```

Comment:
```suggestion
			fgettext("Renderer:") .. "\n" ..
```

Reply:
Also does this really need a newline?

Reply:
On small screens this otherwise insects with the credits page, hence I wanted to keep it limited by using Y space instead,